app/models/user.rb

```diff
@@ -6,10 +6,10 @@ class User < ActiveRecord::Base
 
   has_attached_file :avatar, :styles => { :medium => "300x300>", :avatar =>  "150x150#", :thumb => "100x100>" }, :default_url => "blank_avatar.png"
   validates_attachment_content_type :avatar, :content_type => /\Aimage\/.*\Z/, reject_if: :blank
+
   validates :first_name, presence: true
   validates :last_name, presence: true
   validates :email, presence: true
-  validates :password, presence: true, confirmation: true
-  validates :password_confirmation, presence: true
+  validates :password, presence: true
```

Comment:
If you are only validating the presence of attributes, combine all the attributes and validate the presence as a whole, validates :first_name, :last_name, :email, :password, presence: true.